src/nvim/eval/typval_encode.h

```diff
@@ -154,6 +154,7 @@ typedef enum {
 /// Structure representing current VimL to messagepack conversion state
 typedef struct {
   MPConvStackValType type;  ///< Type of the stack entry.
+  typval_T *tv;  ///< Currently converted typval_T.
```

Comment:
Where is that new member used (other than for writing)? Could it replace member `data`?